packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -140,7 +146,7 @@ test.describe('Participant pane tests', async () => {
     const longPaul = defaultMockRemoteParticipant(
       'I have a really really really really long name. Trust me you dont wanna know.'
     );
-    longPaul.state = 'Connecting';
+    longPaul.state = 'Ringing';
```

Comment:
Test no longer matches test name and the long participant is not truncated:
![image](https://user-images.githubusercontent.com/2684369/211967589-2ed5197c-3844-4ab4-95ad-db61b65ebe2c.png)

Reply:
this is a old image that should be deleted. this test was changed to be for 1:N calling instead of PSTN